client/src/App.js

```diff
@@ -22,7 +22,8 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/" component={MovieList} />
+        <Route path="/movies/:id" component={Movie} />
```

Comment:
This works, but it is better to use render than component. So:
render={props => <Movie {...props} />} instead of component={Movie}
It's technical, but explanation here:
https://stackoverflow.com/questions/48150567/react-router-difference-between-component-and-render
TLDR - component creates new component each time, render does not. Render is faster.